src/components/CreateMenu.js

```diff
@@ -49,7 +49,7 @@ class CreateMenu extends PureComponent {
             [CONST.MENU_ITEM_KEYS.NEW_CHAT]: {
                 icon: ChatBubble,
                 text: 'New Chat',
-                onSelected: () => Navigation.navigate(ROUTES.NEW_CHAT),
+                onSelected: () => Navigation.navigate(ROUTES.IOU_BILL),
```

Comment:
Is this just testing? Or do you mean to change this?

Reply:
I was just meaning to test this in iPhone/Android. I completely forgot to remove it.